code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -245,11 +245,9 @@
 /obj/vehicle/sealed/mecha/atom_destruction()
 	loc.assume_air(cabin_air)
 	for(var/mob/living/occupant as anything in occupants)
-		if(isAI(occupant))
-			occupant.gib() //No wreck, no AI to recover
-			continue
 		mob_exit(occupant, FALSE, TRUE)
-		occupant.SetSleeping(destruction_sleep_duration)
+		if(iscarbon(occupant))
+			occupant.SetSleeping(destruction_sleep_duration)
```

Comment:
Wouldn't it be cleaner to make posibrain SetSleeping() work as expected? i.e. they wake up from their 'sleep' after a bit

Reply:
Yeah I wish, I've tried figuring out why it's like this for a few hours, I just can't.